eth/gasprice/gasprice.go

```diff
@@ -218,6 +218,16 @@ func (oracle *Oracle) SuggestTipCap(ctx context.Context) (*big.Int, error) {
 			exp++
 			number--
 		}
+
+		// if block gas usage is less than 60%, set block's gasPrice as defaultGasPrice
+		if res.gasLimit > 0 && res.gasUsed*10/res.gasLimit < 6 {
+			defaultGasPrice := new(big.Int).Mul(oracle.startPrice, big.NewInt(2))
+			for i, value := range res.values {
+				if value.Cmp(defaultGasPrice) > 0 {
+					res.values[i] = defaultGasPrice
+				}
+			}
+		}
```

Comment:
Maybe it is better to move this block into "here" 
if len(res.values) == 0 {
} else {
    "here"
}